### DISCUSSION TITLE
Aggregate values also for filesystem

### DISCUSSION THREAD

**Discussion by aruku** [GoogleChrome/lighthouse-ci#577] [Q&A]
I just started using LHCI and I noticed that when the results are uploaded to the server, the values seem to be an aggregate (the report that gets uploaded is that with the median value, looks like).
Is there a way to do this when _uploading_ the reports to the filesystem? Meaning, only one report gets chosen.

Thank you.

**Comment by patrickhulce** [comment]
Any Lighthouse report you look at will always be just one report, they're not merged. The values that you see that are aggregated are the statistics on the dashboard. If you don't want more than one report, you can limit the number of runs to just one during collection with `lhci collect -n 1`

Would you mind explaining why it's a problem that there's more data available to look at? Seems like there's a different UX issue at play here if having more data is this problematic.

**Reply by aruku** [reply]
I think I chose a poor title for the issue. What I meant was, the same way that the dashboard aggregates the values and keeps the median, it would be helpful to only keep the report with the median values when _uploading_ them to the filesystem.
At least for consistency (same values on the dashboard and on the files), and that way you don't have to look through different files trying to find the right one in case you need more data from the most useful report.

**Reply by patrickhulce** [reply]
Ah I think I understand now. That's what the `manifest.json` file is for, so you can find the one "representative report" for each URL using the [`isRepresentativeRun` flag](https://github.com/GoogleChrome/lighthouse-ci/blob/4f9bcdc9acd99e46d089b0e13d1ee241b8b298f4/packages/cli/src/upload/upload.js#L560) in the manifest.

If you don't want the other files there, it's simple enough to remove them.
```js
const fs = require('fs');
const manifest = require('./manifest.json');
for (const entry of manifest) {
  if (entry.isRepresentativeRun) continue;
  fs.unlinkSync(entry.htmlPath);
  fs.unlinkSync(entry.jsonPath);
}
